subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/parameters/BuildProgressListenerAdapter.java

```diff
@@ -154,6 +169,23 @@ private StartEvent startedEvent(InternalOperationStartedProgressEvent event) {
         return new DefaultStartEvent(event.getEventTime(), event.getDisplayName(), descriptor);
     }
 
+    private ProgressEvent statusEvent(InternalOperationStatusProgressEvent event) {
```

Comment:
Do we need this yet? Certainly at some point (hopefully soon) we want to send status events from the daemon, but for this PR the only status events are coming from the client side.